Backend/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "inotes-backend",
+  "version": "1.0.0",
+  "description": "Your Notes on the Cloud",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "serve": "nodemon index.js "
+  },
+  "author": "",
```

Comment:
kindly update authors
[
    "Himanshu himanshu007-creator",
    "Hridhima Hridhima",
  ]

Reply:
Thanks for incorporating these changes @Hridhima  😄